common/base/tests/it/string_func.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_base::*;
+use common_exception::Result;
+
+#[test]
+fn test_progress() -> Result<()> {
```

Comment:
How about adding the slash case?
@drmingdrmer expected your suggestions on the escape case :)

Reply:
@BohuTANG 
Looks good to me.
And it does not look like affecting me at all. 🤔

Reply:
@BohuTANG Thanks for your advise. I have changed the test case.